logstash-core/lib/logstash/java_pipeline.rb

```diff
@@ -260,16 +260,11 @@ def start_workers
   end
 
   def resolve_cluster_uuids
-    cluster_uuids = []
-    outputs.each do |output|
-      if LogStash::SETTINGS.registered?(output.id + ".cluster_uuid")
-        cluster_uuid = LogStash::SETTINGS.get(output.id + ".cluster_uuid")
-        unless cluster_uuids.include? cluster_uuid
-          cluster_uuids.push(cluster_uuid)
-        end
+    outputs.each_with_object(Set.new) do |output, cluster_uuids|
```

Comment:
Any logic in `java_pipeline.rb`, the version of the pipeline used with the Java execution engine, will need to be replicated in `pipeline.rb` for the Ruby execution engine. While the Java execution engine is the default execution as of 7.0, users can still choose to use the Ruby execution engine.

Reply:
Ah, I didn't know that. Thanks! I have copied it over.